### PR TITLE
Submission for the assignment on Python_Data_Engineering

### PR DESCRIPTION
**Pull request by Harsh-pandya**
Kindly check the pull request

### CODE REVIEW

File: Layout_table.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import  Table, Column, Integer, String,Date,Time,Float
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+class table_structure(Base):
```
Comment (dwividha) [inline Layout_table.py:6]: There needs to be 2 separate tables:

-  for realtime bike availability of  bikepoints
- for historical bike availability of bikepoints 

With this structure, you'll only be able to store the realtime availability

File: Layout_table.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    __tablename__='bikepoint_table'
+    bike_id = Column(String(30),primary_key=True)
+    name_of_loaction=Column(String(100))
```
Comment (dwividha) [inline Layout_table.py]: possible typo :)

File: fetch_data_from_database.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    
+def fetch(sess,locate):
+    list_data = locate.split()
+    final_str = str()
+    for i in range(len(list_data)):
+        list_data[i] =list_data[i].capitalize()
+        final_str+=list_data[i]+' '
+    final_str+='%'
+    result = sess.query(table_structure).filter(table_structure.name_of_loaction.like(final_str)).all()
+    return result[0].__dict__
```
Comment (dwividha) [inline fetch_data_from_database.py:29]: I like the idea of keeping the SQLAlchemy related objects inside this function and returning a dict instead

**Review by dwividha — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,34 @@
...
+ * ```setx User_name "Enter Your Database user name"```
+ * ```setx Password "Enter Your Database Password of that user"```
+ * ```setx Port "Enter Your Database Port Number"```
+ * ```setx Db_name "Enter Name of Database of your choice"```
+
+* 3) Run the file named ``` Create_Database.py``` it will create a database if it dose not exist 
+
+* 4) Run the script named ```Create_table_and_intialize.py``` it will create the table if it dose not exist and intialize with values
+
+	* To Schedule the python script to run at every 5 minutes in windows we use Task Scheduler
```
Comment (dwividha) [inline README.md]: Try doing this using [python-crontab](https://pypi.org/project/python-crontab/)?

File: Layout_table.py (hunk 1 above)
Comment (Harsh-pandya) [reply Layout_table.py:6]: Thank you @dashu2410 sir  for the review  . I would submit the suggested update in two to three days.
Sir can you please give insight on  historical bike availability of bikepoints whether to store all previous data in single table or store data in separate table with update of realtime bike availabilty of bikepoint table.

**Comment by Harsh-pandya** [comment]
Thank you @dashu2410  sir for all your suggestion . I have made all updates suggested . Can you please review it.

**Pull request closed without merging**